src/utils.rs

```diff
@@ -64,3 +69,20 @@ pub fn get_human_readable_timestamp(duration: Duration) -> String {
 pub fn get_full_username(user: &User) -> String {
     format!("{}#{:04}", user.name, user.discriminator)
 }
+
+pub fn get_prefixes() -> serde_json::Value {
+    let data = fs::read_to_string("prefixes.json").unwrap_or_else(|error| {
+        if error.kind() == ErrorKind::NotFound {
+            let mut f = File::create("prefixes.json").unwrap_or_else(|error| {
+                panic!("Problem creating the file: {:?}", error);
+            });
+            f.write_all("{}".as_bytes())
+                .expect("Unable to write to file");
+            "{}".to_string()
```

Comment:
Why this? It is not returned

Reply:
We need to initialize the file as a valid json, otherwise it may fail later on.

Reply:
This: https://github.com/afonsojramos/parrot/blob/afonso/prefix-change/src/utils.rs#L81
is not returning it.

This is: https://github.com/afonsojramos/parrot/blob/afonso/prefix-change/src/utils.rs#L87

Reply:
Actually, it is returning the string to the `data` variable.